client/src/App.jsx

```diff
@@ -24,6 +25,7 @@ const App = () => (
 					<Route path="/drafts" element={<Authenticated adminOnly />}>
```

Comment:
Suggestion: In order to turn out the code more clear and consistent, adding "adminOnly={true}" would be nice.